### DISCUSSION TITLE
Just wanted to say thanks

### DISCUSSION THREAD

**Discussion by jhimes144** [AvaloniaUI/Avalonia#11883] [General]
I've been developing a massive project with Avalonia and it all started with version 10. I've been following the previews as they came out, updating our code base. Its been an amazing journey. I've been making apps in WPF for years, and this really is the successor. Its like WPF got married with Miss Modern Paradigm and they had a prodigy son called Avalonia with its amazing style library, performance, and power.

This is a great UI framework. Thank you.

**Comment by MikeCodesDotNET** [comment]
Thank you for the lovely words! 

I'd love to learn more about what you've been building, maybe something we could include on the showcase?

**Reply by jhimes144** [reply]
When it is released I would love for it to be included in the showcase! The project is a entertainment lighting control system designed to compete with leading lighting consoles - its been over two years in development with the first version made in WPF over the course of 3 years. Enttec - one of your sponsors, is also a player in our industry.

Our project includes a primary desktop app which runs on Mac or Windows and a mobile app that we plan on having released for Android, iPad, Windows, and Linux.

**Reply by MikeCodesDotNET** [reply]
Now I'm even more interested! 

I used to work with the old Flying Pig guys ([Carallon](https://www.carallon.com/) / [Pharos](https://www.pharoscontrols.com/)), so lighting control is one of my passions! 

Building a lighting controller is how I got into Avalonia in the first place! Very excited to see what you've been working on. If you need any testers, I'd love to give it a try. I've a license for Capture and a hand full of DMX devices lying around the office.

**Reply by jhimes144** [reply]
@MikeCodesDotNET 

Sorry for the late delay. I was actually out at a festival giving the alpha a test run at two different stages for 2 days.

The project is called "Windchime". It is the work between myself and one other person. Here is a video of it controlling the lights for the headliner that day - a US metalcore band called Memphis Mayfire, using a giant touchscreen and mini pc.

https://drive.google.com/file/d/1ITpEz9f7fFAAhtUu6C5EFWjPRZmgOdXX/view?usp=sharing

Here is a picture of the UI, particularly the section you use to busk a show, which is fully customized by the user.
![wCapture7](https://github.com/AvaloniaUI/Avalonia/assets/20263191/05bd5f6e-e3d2-4b93-93f8-1e64dce6486e)

And another picture: There is still a lot we want to do with this screen.
![wCapture6](https://github.com/AvaloniaUI/Avalonia/assets/20263191/2d8cf290-4f90-4f22-8fda-45d177e7f722)

It would be an honor to have you as a tester. I'm not sure how we could exchange emails on here, I'd like to keep mine private :). Windchime is designed to compete at the level of big name consoles while being extremely intuitive and easy to operate - doing away with many concepts and paradigms that have followed traditional consoles from the 80s and 90s.

The ability for me and my partner to focus on our innovation and creativity I owe solely to the power of Avalonia.

**Reply by MikeCodesDotNET** [reply]
It looks great! 

I spoke with @pixsperdavid not too long ago about how lighting controls systems havn't changed much since the Hog2, its a big frustration for me.  

Anyway, I'd love to try your app. You can contact me via email at mike@avaloniaui.net
